### PR TITLE
fix FluxCD URLs

### LINKED ISSUE #229: FluxCD postinstall shows wrong bash installation URL
This issue is a:
* [X] Bug Report
* [ ] Enhancement suggestion
* [ ] Feature request

If the issue concerns an existing Marketplace application, please tag the maintainer of the application (found in `manifest.yaml`) below:

@stefanprodan 

## Bug Reports

The [PostInstall report](https://github.com/civo/kubernetes-marketplace/blob/master/flux/post_install.md#upgrade-instruction) mentions a old link to the FluxCD installation website for Bash.
```sh
curl -s https://toolkit.fluxcd.io/install.sh | sudo bash
```

Actually it currently is:
```sh
curl -Is https://toolkit.fluxcd.io/install.sh

HTTP/2 301
cache-control: public, max-age=0, must-revalidate
content-length: 44
content-type: text/plain; charset=utf-8
date: Thu, 15 Jul 2021 07:55:15 GMT
age: 294939
server: Netlify
location: https://fluxcd.io/install.sh
```

This leads to a weird behavior if you just copy&paste the command

EDIT:
As it seems the general domain "toolkit.fluxcd.io" changed to "fluxcd.io" throughout the rest of the documentation.

### PR DESCRIPTION
**Pull request by dirsigler**
If your pull request concerns an existing Marketplace application, please make sure you have:

* [X] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [X] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [X] Tested that the application works with the proposed updates applied (including a screenshot)

This is just a change which fixes the URL of `toolkit.fluxcd.io` to just `fluxcd.io`.

Test:
```sh
curl -Is https://fluxcd.io/install.sh
HTTP/2 200
accept-ranges: bytes
cache-control: public, max-age=0, must-revalidate
content-length: 5600
content-type: application/x-shellscript
date: Sun, 25 Jul 2021 18:13:47 GMT
strict-transport-security: max-age=31536000
age: 3260
etag: "910be6a0f291257fc5e2534e74d4b28b-ssl"
server: Netlify
x-nf-request-id: 01FBFH2FF51VCK26W15655FEGS
```

@stefanprodan this fixes #229 .

### CODE REVIEW

File: flux/post_install.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Flux
 
 Flux is a Continuous Delivery solution for Kubernetes.
-Flux is constructed with the [GitOps Toolkit](https://toolkit.fluxcd.io/components/),
+Flux is constructed with the [GitOps Toolkit](https://fluxcd.io/components/),
```
Comment (stefanprodan) [inline flux/post_install.md:4]:
```suggestion
Flux is constructed with the [GitOps Toolkit](https://fluxcd.io/docs/components/),
```

File: flux/post_install.md [hunk 2]
```diff
@@ -14,20 +14,20 @@ Flux is a [CNCF](https://cncf.io) project made for:
 
 ### Usage instruction
 
-To get started with Flux, [browse the documentation](https://toolkit.fluxcd.io) or try the following guides:
+To get started with Flux, [browse the documentation](https://fluxcd.io) or try the following guides:
 
-- [Get started with Flux](https://toolkit.fluxcd.io/get-started/)
+- [Get started with Flux](https://fluxcd.io/get-started/)
```
Comment (stefanprodan) [inline flux/post_install.md:19]:
```suggestion
- [Get started with Flux](https://fluxcd.io/docs/get-started/)
```

File: flux/post_install.md [hunk 3]
```diff
@@ -14,20 +14,20 @@ Flux is a [CNCF](https://cncf.io) project made for:
...
 - [GitOps workflow for multi-env deployments with Flux, Kustomize and Helm](https://github.com/fluxcd/flux2-kustomize-helm-example)
-- [Manage Kubernetes secrets with Mozilla SOPS](https://toolkit.fluxcd.io/guides/mozilla-sops/)
-- [Automate container image updates to Git](https://toolkit.fluxcd.io/guides/image-update/)
+- [Manage Kubernetes secrets with Mozilla SOPS](https://fluxcd.io/guides/mozilla-sops/)
```
Comment (stefanprodan) [inline flux/post_install.md:21]:
```suggestion
- [Manage Kubernetes secrets with Mozilla SOPS](https://fluxcd.io/docs/guides/mozilla-sops/)
```

File: flux/post_install.md [hunk 4]
```diff
@@ -14,20 +14,20 @@ Flux is a [CNCF](https://cncf.io) project made for:
...
+- [Automate container image updates to Git](https://fluxcd.io/guides/image-update/)
```
Comment (stefanprodan) [inline flux/post_install.md:22]:
```suggestion
- [Automate container image updates to Git](https://fluxcd.io/docs/guides/image-update/)
```

**Review by stefanprodan — CHANGES_REQUESTED** [review]

File: flux/post_install.md (hunk 1 above)
Comment (dirsigler) [reply flux/post_install.md]: Good catch 😌

**Review by stefanprodan — APPROVED** [review]
LGTM 

Thanks @dirsigler

**Comment by saiyam1814** [comment]
awesome thank you both :)

**Pull request merged**